fml/raster_thread_merger.cc

```diff
@@ -48,11 +62,17 @@ void RasterThreadMerger::ExtendLeaseTo(size_t lease_term) {
 }
 
 bool RasterThreadMerger::IsMerged() const {
-  return is_merged_;
+  return is_merged_.load();
```

Comment:
`load` and `store` are not necessary as the operators are overloaded on `std::atomic<foo>`.

Reply:
done